src/plugins/home/public/application/components/tutorial_directory.js

```diff
@@ -93,11 +93,12 @@ class TutorialDirectoryUi extends React.Component {
 
   async componentDidMount() {
     this._isMounted = true;
+    const { chrome, homeLink } = getServices();
```

Comment:
Seems homeLink only be used in this component, put `homeLink` in services would a little bit tricky. How about add a optional `homeLink` property in `TutorialDirectoryUi` and passed from  `renderTutorialDirectory`.

Reply:
Sure, updated that.